### ISSUE TITLE
Doesn't work on Firefox. simplebar-scroll-content has height="0"

### ISSUE THREAD

**Issue by ivanceban** [Grsmto/simplebar#36]
Hi! 
Thank you for nice plugin! 
I'm using simplebar and it works fine on all browsers except Firefox. 
On Firefox simplebar-scroll-content and simplebar-scrollbar have height dynamically set to "0". So no content is displayed. 
Maybe I'm doing something wrong. But it works fine everywhere except Firefox. So I'm confused, what is wrong with Firefox?

This is how it looks on Chrome:
![screen shot 2016-09-08 at 4 38 14 pm](https://cloud.githubusercontent.com/assets/13540144/18351414/c76f6744-75e2-11e6-995e-6a1c988cdc12.png)

And this is what I see on Firefox:
![screen shot 2016-09-08 at 4 38 22 pm](https://cloud.githubusercontent.com/assets/13540144/18351438/d853ff70-75e2-11e6-8b6f-337c54496190.png)

Any idea?

Thank you

**Comment by Grsmto** [comment]
Hi,
What version/os of Firefox are you running?
Does it work for you on the demos? http://grsmto.github.io/simplebar/demos/
Thank you!

**Comment by Grsmto** [comment]
Ah, also, I see you use Simplebar on the `body` element. I never tried to use it this way but you need to know that the parent element needs to have a defined height.

However I don't recommend to use Simplebar for the `body` element as it will significantly slow down the general performances of you website (the `overflow: auto` scroll is slower than the scroll of the window).

**Comment by ivanceban** [comment]
Thank you, for quick response, Adrien.
I'm on OS X El Captain v10.11.5 and Firefox v48.0.
All demos work in Firefox. And you are right, the problem was in the parents defined height.
When I set .simplebar {height: 100vh} for main (page) scrollbar, it worked. 

![screen shot 2016-09-09 at 10 06 04 am](https://cloud.githubusercontent.com/assets/13540144/18378380/53b9f330-7675-11e6-899b-771efba6893d.png)

But I'm afraid that not all browsers understand vh units and I will get problems with crossbrowsering. (I had problems with "vh" on iOS devices for example). Am I doing it right or there is a more clever way to use Simplebar as page scrollbar?

And one more thing: I use simplebar also in dropdowns (select2 plugin). So for dropdown I can't set fixed height (if content is less then container height it looks bad). The only way that I found is to set fixed height to container and then measuring the height of content in jQuery and overwrite the height of dropdown container. 
![screen shot 2016-09-09 at 10 14 50 am](https://cloud.githubusercontent.com/assets/13540144/18378582/7c1bb1a0-7676-11e6-864a-4567b0ae9492.png)

![screen shot 2016-09-09 at 10 16 49 am](https://cloud.githubusercontent.com/assets/13540144/18378594/8cb5cc9e-7676-11e6-84c5-b9d0b6373d65.png)


I'm not sure that this is good decision, but I don't see any other. And only in this way it works. 
Maybe you have a better vision?
Thank you,
Ivan
